src/views/LandingView.vue

```diff
@@ -97,6 +95,11 @@
                 <div>
                     <h2>Have you already seen <br><span>Song Treasures</span>?</h2>
                     <p>Go to <a href="https://songtreasures.org/">songtreasures.org</a></p>
+                    <p :style="{'font-weight': 'normal'}">Song Treasures is another product on the Treasures platform. 
+                        While Wisdom Treasure provides you with edifying words, 
+                        the Song Treasures is a collection of spiritual/Christian music, songbooks and 
```

Comment:
Kanskje uten **the**?
Altså _...edifying words, Song Treasures is a collection of spiritual/Christian music..._